source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -365,6 +367,9 @@ namespace PlayFab
 
     bool PlayFabWinHttpPlugin::GetBinaryPayload(CallRequestContainer& reqContainer, LPVOID& payload, DWORD& payloadSize) const
     {
+        UNREFERENCED_PARAMETER(reqContainer);
```

Comment:
Perfect, I remember seeing some errors on the mac regarding unreferenced params.